### ISSUE TITLE
request:

### ISSUE THREAD

**Issue by r3k2** [Binject/awesome-go-security#4]
Hello can we clone this on gitlab and update it? if you guys dont have time I can do it and help I myself build all my tools for pentest/CTF/systems on GO . why? because lot of us that try to keep things ethical and open try to avoid gitlab so that way we can make the project more inclusive for everyone's ethics and uses.

**Comment by ahhh** [comment]
I think thats a great idea, can you just keep both up to date? also a link to the other project would be helpful. Would be even cooler if we made gitlab clones of all the projects it references

**Comment by ReK2Fernandez** [comment]
@ahhh  Hello, cool I have forked the project and created a sync with gitlab for now since I do not have creds or anything to do it directly from Binjet. 
https://gitlab.com/HispaGatos/awesome-go-security
haha to make clones of everything we will have to ask each of them first just like I did with you guys :)  maybe something we can do moving forward.
And yeah I will be glad to keep updating it if I can have access to the repo.
Thanks
